### PR TITLE
97.edit and view photo for vibe event

### LINKED ISSUE #97: Edit and View Photo for Vibe Event
Need to show an image onLoad for the Edit Vibe activity and when Viewing

### PR DESCRIPTION
**Pull request by jacobwong98**
[//]: # (These are comments and are used for reference and will not show up in the PR)

Description: 

- Display image when viewing/editing a VibeEvent

[//]: # (Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change)

[//]: # (automatically closes given issue number)
Closes #97   

Proof of Concept

image
image

[//]: # (Attach Image of feature added or fix applied and working)
[//]: # (Please provide images of tests passing)

Checklist

[//]: # (Add an x between square brackets to mark item as done)

- [x] Follows style guidelines and is **properly commented**
- [x] Code changes generate no new warnings
- [ ] Testing (Unit and/or Intent)
- [x] Documentation (UML, Wiki, etc.)

### CODE REVIEW

File: app/src/main/java/com/example/vybe/AddEditVibeActivity.java [hunk 1]
```diff
@@ -304,4 +310,27 @@ public String formatDateTime(LocalDateTime dateTime) {
...
+        StorageReference storageRef = FirebaseStorage.getInstance().getReference();
+
+        // Get the path to the image
+        StorageReference imageRef = storageRef.child(imagePath);
+
+        // Get the download URL for Glide
+        imageRef.getDownloadUrl().addOnSuccessListener(new OnSuccessListener<Uri>() {
+            @Override
+            public void onSuccess(Uri uri) {
+                Glide.with(getApplicationContext() /* context */)
```
Comment (raysarinas) [inline app/src/main/java/com/example/vybe/AddEditVibeActivity.java]: What up with the `/* context */` comment

File: app/src/main/java/com/example/vybe/AddEditVibeActivity.java [hunk 2]
```diff
@@ -304,4 +310,27 @@ public String formatDateTime(LocalDateTime dateTime) {
...
 
+    /**
+     * This will load an image from Firebase Storage into an ImageView
+     * @param imageView
+     *      The destination ImageView
+     * @param imagePath
+     *      Path to the image in Firebase Storage
+     */
+    public void loadImageFirebase(ImageView imageView, String imagePath){
+        StorageReference storageRef = FirebaseStorage.getInstance().getReference();
```
Comment (raysarinas) [inline app/src/main/java/com/example/vybe/AddEditVibeActivity.java]: are these names okay? might get confusing

File: app/build.gradle [hunk 3]
```diff
@@ -41,6 +41,9 @@ dependencies {
     androidTestImplementation 'com.jayway.android.robotium:robotium-solo:5.3.1'
     implementation 'com.google.firebase:firebase-auth:19.1.0'
     implementation 'com.google.android.gms:play-services-maps:17.0.0'
+    implementation 'com.github.bumptech.glide:glide:4.10.0'
```
Comment (raysarinas) [inline app/build.gradle]: Make documentation on Glide! Please

File: app/src/main/java/com/example/vybe/AddEditVibeActivity.java (hunk 1 above)
Comment (jacobwong98) [reply app/src/main/java/com/example/vybe/AddEditVibeActivity.java]: Removed

File: app/build.gradle (hunk 3 above)
Comment (jacobwong98) [reply app/build.gradle]: Will update documentation

File: app/src/main/java/com/example/vybe/AddEditVibeActivity.java (hunk 2 above)
Comment (jacobwong98) [reply app/src/main/java/com/example/vybe/AddEditVibeActivity.java]: It should be fine to leave them like that. We can change if it does get confusing

**Pull request merged**